iOSDFULibrary/Classes/Implementation/SecureDFU/Peripheral/SecureDFUPeripheral.swift

```diff
@@ -65,7 +73,7 @@ internal class SecureDFUPeripheral : BaseCommonDFUPeripheral<SecureDFUExecutor,
     func jumpToBootloader() {
         jumpingToBootloader = true
         newAddressExpected = dfuService!.newAddressExpected
-        dfuService!.jumpToBootloaderMode(
+        dfuService!.jumpToBootloaderMode(dontSetAdvertisingName: disableSettingAlternativeAdvertisingName,
```

Comment:
Same here, `dontSetAdvertisingNAme: disableAlretnativeAdvertisingName` is a bit confusing and unreadable in a way.

can it be renamed to :

`jupToBootloaderMode(withAlternativeAdvertisingName: alternativeAdvertisingFeatureEnabled, ...)`

or something similar ?